tests/tar.test.php

```diff
@@ -186,7 +186,38 @@ public function test_dogfood()
             unlink($archive);
         }
     }
+	
+	/**
```

Comment:
indentation seem wrong again. please make sure you use 4 spaces for indentation. not tabs!